src/elements/common/logger/__tests__/Logger.test.js

```diff
@@ -118,6 +118,31 @@ describe('elements/common/logger/Logger', () => {
         });
     });
 
+    describe('handleDataReadyMetric()', () => {
+        const END = 'end';
+        const START = 'start';
+        const data = {
+            foo: 'bar',
+            endMarkName: END,
+            startMarkName: START,
+        };
+        let instance;
```

Comment:
Is this needed if we only have the one test? Would something like `jest.spyOn(wrapper.instance(), 'logUniqueMetric');` work?